HPXMLtoOpenStudio/resources/constructions.rb

```diff
@@ -2032,6 +2035,14 @@ def self.create_kiva_slab_foundation(model, int_horiz_r, int_horiz_width, int_ve
       foundation.setExteriorVerticalInsulationDepth(UnitConversions.convert(ext_vert_depth, 'ft', 'm'))
     end
 
+    # Exterior horizontal insulation
+    if (ext_horiz_r > 0) && (ext_horiz_depth > 0) && (ext_horiz_width > 0)
+      ext_horiz_mat = create_insulation_material(model, 'exterior horizontal ins', ext_horiz_r)
+      foundation.setExteriorHorizontalInsulationMaterial(ext_horiz_mat)
+      foundation.setExteriorHorizontalInsulationDepth(UnitConversions.convert(ext_horiz_depth, 'ft', 'm'))
+      foundation.setExteriorHorizontalInsulationWidth(ext_horiz_width)
```

Comment:
This is missing a unit conversion. Confirmed the bug by looking at the IDF. I will fix this.